pages/api/youtube.js

```diff
@@ -0,0 +1,55 @@
+import fetch from 'node-fetch'
+import config from 'lib/config'
+
+// const {asSeconds} = require('pomeranian-durations')
+
+const {YOUTUBE_API_KEY: key} = config
+
+export default async function handler(req, res) {
+	const ytid = req.query.ytid
+
+	try {
+		// Make sure we have what we need.
+		if (!key) throw new Error('A YOUTUBE_KEY in your .env file is required')
```

Comment:
`key` vs `youtubeApiKey` vs `apiKey`? maybe a middle way (key/value?!)
also semantics of the API KEY itself..